packages/create-webstone-app/src/tasks/2-setup-project/plugin.ts

```diff
@@ -55,6 +55,24 @@ export const createCliNamespace = (ctx: Ctx) => {
       getAppName(ctx.appDir)
     )
   );
+  fs.renameSync(
```

Comment:
forgot to rename the extensions namespace

Reply:
Moved this to #283.